File: aepc-angular-front/src/app/shared/models/project/project.model.ts
import { State } from './project.state';
import { Priority } from "./project.priority";

export class Project {
  projectId!: string;
  name!: string;
  description!: string;
  priority!: Priority;
  state!: State;
  createdDate!: Date;
  employeeId!: string;
  companyId!: string;
}

File: aepc-back-bs-ms-modules/aepc-api-spring-secur-oauth2-kafka-avro-bs-ms-employee/src/main/java/com/placide/k8skafkaavroaepccleanarchibsmicrosemployee/infra/adapters/input/feignclients/models/ProjectModel.java
package com.placide.k8skafkaavroaepccleanarchibsmicrosemployee.infra.adapters.input.feignclients.models;

import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

@Builder
@Setter
@Getter
public class ProjectModel {
    private String projectId;
    private String name;
    private String description;
    private int priority;
    private String state;
    private String createdDate;
    private String employeeId;
}
